examples/sample-test-app1.yaml

```diff
@@ -106,3 +106,36 @@ spec:
     persistentVolumeClaim:
       claimName: pv1-3
   restartPolicy: OnFailure
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: pv1-4
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Block
+  storageClassName: kadalu.storage-pool-1
+  resources:
+    requests:
+      storage: 500Mi
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod1-4
+spec:
+  containers:
+    - name: bash-block-vol
+      image: bash
+      command: ["/usr/local/bin/bash","-c"]
+      args: ["echo sample text > /tmp/original && dd if=/tmp/original of=/dev/xvda && dd if=/dev/xvda of=/tmp/copy bs=1 count=12 && cat /tmp/{original,copy} && diff <(cat /tmp/original) <(cat /tmp/copy) && echo all good"]
```

Comment:
@amarts Is this test sufficient to say it is actually a device file or a test required to create a fs and mount?

Reply:
- I'm facing issues in my local setup and on CI as well for creating and mounting a file system on this PV disk
- Docker in docker or similar setups seems not to be working quite well with mount namespace isolation/permissions
- My thought process for docs is
1. Use `pv_type: Block` if user wants to have a fs mounted on block file/device
2. Use `volume_mode: Block` if user want to have access to a block file/device

Reply:
> Use volume_mode: Block if user want to have access to a block file/device

What are the usecases for this? Ie, a block device without mount.

Reply:
- I'm thinking of databases which can use raw disk but not quite sure though
- Nevertheless, if we want to be sure, we need to deploy containers created from this PR on actual node